packages/transaction-controller/src/TransactionController.ts

```diff
@@ -957,8 +978,12 @@ export class TransactionController extends BaseController<
       this.markNonceDuplicatesDropped(transactionId);
 
       // Update external provided transaction with updated gas values and confirmed status.
-      this.updateTransaction(transactionMeta);
+      this.updateTransaction(
+        transactionMeta,
+        'TransactionController:confirmExternalTransaction - Add external transaction',
+      );
     } catch (error) {
+      console.log('asddsaadsads');
```

Comment:
I think this was added by mistake.

Reply:
Fixed in https://github.com/MetaMask/core/pull/1657/commits/4389be95516ee0b210a7d455fa86d573e1a01909